generate_documentation_qmd/README.md

```diff
@@ -0,0 +1,196 @@
+
+* `output_dir`: 'Output folder to write the qmd files to. If you wish to output to another repository, you can use [checkout](https://github.com/marketplace/actions/checkout) to fetch that repo and write to repository clone.'
+
+* `dest_path` - _optional_. Template for the location of the qmd files. Can contain `{type}`, `{namespace}` and `{name}` wildcards to create a custom folder hierarchy for storing the output for different components. Default: `"{type}s/{namespace}/{name}.qmd"`
+
+* `git_tag` - _optional_. A git tag or branch. Used to create links to the source code in the generated documentation. Default: tag of the current branch. Ex: `0.2.0`
+
+* `git_repo` - _optional_. Github repository. Used to create links to the source code in the generated documentation.
+Must use format "owner/repo-name". Will default to the repo where this action is being run for. Ex: `openpipelines-bio/openpipeline`
+
+* `write_index` - _optional_. Whether or not to also create a quarto index file that joins the different pages for the components and workflows. Options are `'true'` or `'false'`.
+
+* `token` - Personal access token (PAT) used to fetch the viash tools repository.
+
+* `tools_version` - _optional_. Release of Viash tools to use. Will use the latest release by default. Ex: `main_build`
+
+
+## Examples
+
+```yaml
+name: demo of generate_documentation_qmd
+
+on:
+  pull_request:
+
+jobs:
+  demo:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: viash-io/viash-actions/setup@v1.0.0
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v34
+
+    - name: List all changed files
+      run: |
+        for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+          echo "$file was changed"
+        done
+    
+    - uses: actions/checkout@v3
+      with:
+        repository: openpipelines-bio/website
+        path: openpipelines-website
+        
+    - uses: actions/checkout@v3
+      with:
+        repository: openpipelines-bio/openpipeline
+        path: openpipelines
+        ref: "main_build"
+
+    - uses: viash-io/viash-actions/generate_documentation_qmd@generate_documentation_qmd
+      with:
+        input_dir: openpipelines
+        output_dir: openpipelines-bio/website/components
+        dest_path: "{type}s/{namespace}/{name}.qmd"
+        git_repo: openpipelines-bio/openpipelines
+        token: ${{ secrets.GTHB_PAT }}
+        tools_version: main_build
+        
+
+```
+
+# Example template
+
+````
+
+---
+title: "{{functionality.info.name}}"
+namespace: "{{functionality.info.namespace}}"
+description: "{{functionality.info.short_description}}"
+type: "{{info.type}}"
+toc: false
```

Comment:
Nice usage of the quadruple backticks, but maybe we can store the template in a separate file and link to it here?